server/middlewares/substack.ts

```diff
@@ -7,19 +7,23 @@ const client = buildClient()
 
 const substack = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
   const host = req.get('host')
+  let pageUrl = req.query.url as string || ''
```

Comment:
This was initially proposed, but changed later, because anyone would be able to use the server as a proxy and spam